docs/Server-configuration.md

```diff
@@ -4,23 +4,23 @@ Some information about all the possible configurations. Click [here](https://git
 
 ## Options
 
-### Server
+### Required Server Configuration
 
 | Key | Default | Description |
 |-|-|-|
-| **PublicIp** | `127.0.0.1` | This needs to the IPv4 address of the server, what you give to others to connect. You can find the your IPv4 address [here](http://whatismyip.host/). Since 1.2.2 it is also possible to use hostnames instead of IPv4 addresses, these must resolve to a valid IPv4 address. |
-| **PublicPort** | `22023` | The public port of the server, what you give to others to  connect. Usually `22023`. |
+| **PublicIp** | `127.0.0.1` | This needs to the public IPv4 address of the server which you give to others to connect. You can find your IPv4 address [on this website](http://whatismyip.host/). **Unless you are only planning to use Impostor privately, on your local network, you should change this to your public ip. You can still set this to "0.0.0.0" or your private ip (can be obtained by running `ipconfig` in the command prompt) if you are only using Impostor on your local network, or you are using a VPS.** Is also possible to use hostnames instead of IPv4 addresses, which will be resolved to IPv4 addresses. |
```

Comment:
What? You can NOT set this to `0.0.0.0`. `0.0.0.0` does not resolve to anything locally. Entire bold sentence reads very weird.

Reply:
In VPSes it can actually work. Since 0.0.0.0 resolves to all interfaces, if one has the ip set as the public one, there isnt any problem at all. But yes I can remove the bold

Reply:
Are you kidding me..? This would be fine for `ListenIp` but this is `PublicIp`. `PublicIp` is sent to the clients that try to connect to a public game in the game browser. `0.0.0.0` is not a valid PUBLIC ip address.

Reply:
Done removed... (*but it hella works*)